### PR TITLE
chapter1_part1:Install es

### PR DESCRIPTION
**Pull request by autumnbz**
安装es

### CODE REVIEW

File: 010_Intro/10_Installing_ES.asciidoc [hunk 1]
```diff
@@ -1,3 +1,4 @@
+﻿demo
```
Comment (kimichen123) [inline 010_Intro/10_Installing_ES.asciidoc]: fdsafdsa

**Comment by kimichen123** [comment]
LGTM

**Comment by autumnbz** [comment]
LGTM

**Comment by autumnbzs** [comment]
LGTM

**Pull request merged**